app/routes/automated-tools.js

```diff
@@ -72,8 +72,42 @@ export default class AutomatedToolsRoute extends Route {
         category: 'Testing Engine',
         description: 'A accessibility testing engine for JavaScript.',
         link: 'https://github.com/dequelabs/axe-core'
-      },
-      
+      }, {
+        title: 'alfa',
+        category: 'Testing Engine',
+        description: 'Suite of open and standards-based tools for performing reliable accessibility conformance testing at scale.',
+        link: 'https://github.com/Siteimprove/alfa'
+      }, {
+        title: 'AccessLint',
+        category: 'Code Linter (Static)',
+        description: 'AccessLint brings automated web accessibility testing into your development workflow. When a pull request is opened, AccessLint reviews the changes and comments with any new accessibility issues, giving you quick, timely, and targeted feedback, before code goes live.',
+        link: 'https://github.com/marketplace/accesslint'
+      }, {
+        title: 'equal-access',
+        category: 'Browser Extension, Testing Engine, Testing Library',
+        description: 'IBM Equal Access Accessibility Checker contains tools to automate accessibility checking from a browser or in a continuous development/build environment.',
+        link: 'https://github.com/IBMa/equal-access'
+      }, {
+        title: 'JAWS Inspect',
+        category: 'Testing Tool',
+        description: 'A screen reader testing tool that vastly simplifies accessibility and JAWS® compatibility testing by providing a text output of JAWS speech.',
+        link: 'https://www.tpgi.com/arc-platform/jaws-inspect/'
+      }, {
+        title: 'AccessLint/screenreaders',
+        category: 'AT Automation',
```

Comment:
I created a new category for these things that are directly trying to automate AT, since they didn't really fit anywhere else.

Wasn't sure how to name it, or if I should explicitly call out "AT" meaning "Assistive Technology"

Reply:
I think it's fine to assume that our audience knows what AT means for the time being. I can also take a note to add maybe a glossary page or something in the footer for common abbreviations. WDYT?

Reply:
Yeah I think that makes sense. It's presumably as common as a11y now that I think of it